web/docs/03_infra/1.Overview.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation/Infrastructure/Overview" />
+
+# Infrastructure Overview
+
+The FreeBoardGames.org infrastructure is a mixture of frameworks, libaries, tools, and techniques 
+comprising an ecosystem to manage game development, build, deployment, and user feedback relating to enclosing
+content.  It is the culmination of years of effort we're happy to share with our community of game developers,
+contributors, and enthusiast players.
+
+## Core Infra
+
+The following table describes a core elements of FBG Infra:
+
+###todo: break down further between server, web, build, packaging
+
+| What | Description        | Type |
+| -------- | ------------------ | ------- |
+| Node.js       | ES Runtime for executing js   | Native Libary |
```

Comment:
I think it is better to give a link to the library's docs instead of than describing them here.

Reply:
Agreed I think I'll add that as any preamble to the support documents - that being said, there is room for introducing HOW we use these products -- most are peer dependencies (Node, Yarn, etc) and don't always show in meaningful ways in our code artifacts.  I also think this needs additional refinement into 4 or 5 sub categories: Web, Server, Build, Deploy, etc. @richardaum